metadata.yaml

```diff
@@ -2,22 +2,19 @@
 # See LICENSE file for licensing details.
 
 name: pollen
-display-name: Charm Template
-summary: A very short one-line summary of the charm.
+display-name: Pollen
+summary: "Pollen Entropy-as-a-Service"
+maintainers:
+  - launchpad.net/~canonical-is-devops
+issues: https://github.com/canonical/pollen-operator/issues
+source: https://github.com/canonical/pollen-operator
+docs: Yet to be published.
```

Comment:
Set an empty string instead. IDK how this placeholder will impact the publishing action

Reply:
In the case of resolving comments without making changes, please indicate the reason why this won't be changed, I'm wondering the same thing as Arturo